extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/config/config.xml

```diff
@@ -27,4 +27,22 @@
         </parameter>
     </config-description>
 
+    <config-description uri="channel-type:lifx:color">
+        <parameter name="powerOnBrightness" type="integer" min="0" max="100" required="false" unit="%">
+            <label>Power on brightness</label>
+            <description>Brightness level used when switching on the light. Use empty value to leave brightness as is.</description>
+        </parameter>
+        <parameter name="powerOnColor" type="text" pattern="(36[0]|3[0-5][0-9]|[12][0-9][0-9]|[1-9][0-9]|[0-9]),(100|[1-9][0-9]|[0-9]),(100|[1-9][0-9]|[0-9])" required="false">
+            <label>Power on Color</label>
+            <description>Comma separated Hue (0-360), saturation (0-100) and brightness (0-100) levels used when switching on the light. Use empty value to leave it as is.</description>
+        </parameter>
+    </config-description>
+
+    <config-description uri="channel-type:lifx:temperature">
+        <parameter name="powerOnTemperature" type="integer" min="0" max="9000" required="false">
+            <label>Power on temperature</label>
+            <description>Color temperature level used when switching on the light. Both percent (0-100) and kelvin degree (2500-9000) values are supported. Use empty value to leave color temperature as is.</description>
```

Comment:
We do not allow kelvin settings on the channel itself, so I would very much prefer to stick to the same values for the configuration here, i.e. only support 0-100 (which would also make it simpler for UIs to check for valid settings).

Reply:
> We do not allow kelvin settings on the channel itself

@kaikreuzer, what if I introduce separated temperature channel which accept kelvins, is that then ok? Dedicated number channel which have own power on temperature feature in kelvins. Or is you opinion in general that kelvins shouldn't be used. Personally I like the kelvins more as 0-100 is pretty unclear.

Reply:
We decided against Kelvin-channels in all color bulb bindings and use a dimmer channel instead, which represents the "warmth". As this is for all bindings, I would strongly suggest to keep the very same modelling here as well.

Reply:
Maybe lifx binding could show the new age for all bulb bindings ;-)

Seriously speaking, @wborn warn me about this as well, so I will remove the kelvins then.